### ISSUE TITLE
Implement {image} tag

### ISSUE THREAD

**Issue by JackMorganNZ** [uccser/kordac#7]
Implement the existing tag used in the CSFG. The function is found within [`markdownsection.py`](https://github.com/uccser/cs-field-guide/blob/develop/generator/markdownsection.py).

```
[image]
    regex: ^\{image (?P<args>[^\}]*)\}
    function: add_image
```

This tag could also implement the unused `{image set}` tag:

```
[image set]
    regex: \{image-set\}\n(?P<images>(\{image [^\}]*\}\n)+)\{image-set end\}
    function: add_image_set
```

**Comment by hayleyavw** [comment]
fb24586aa98e8432b93c3fdc3c527049c438b833
ba85c39b6c2c5119f15c47aa6bb0e958f0ecb76f
c0f40a04308186e83b40d48a86c10293eb7b5f2a
Was trying to fix the image tag to not remove lines immediately before the tag, this did not need to be fixed as the image tag is a "block" - i.e. there has to be a newline above and below. Rules like this should be included in the docs (#21)

**Comment by hayleyavw** [comment]
Needs to return required files (set of file paths).
